CMakeLists.txt

```diff
@@ -220,14 +220,6 @@ INCLUDE(${CMAKE_ROOT}/Modules/CheckTypeSize.cmake)
 
 CHECK_FUNCTION_EXISTS(posix_memalign HAVE_POSIX_MEMALIGN)
 
-CHECK_INCLUDE_FILE(dlfcn.h HAVE_DLFCN_H)
-CHECK_INCLUDE_FILE(inttypes.h HAVE_INTTYPES_H)
```

Comment:
"inttypes.h" still include in some files under src/einspline. Probably all remove.

Reply:
You are welcome to, I don't have time.

Reply:
I experimented. Used to get intptr_t. Required.

Reply:
Replaced with `<stdint.h>`